opal/views.py

```diff
@@ -299,7 +299,7 @@ def get(self, *args, **kw):
 
 def csrf_failure(request, reason):
     if request.POST:
-        next_url = request.POST.get('next', '/')
+        next_url = request.GET.get('next', '/')
```

Comment:
I completely missed this :)

Reply:
Me too!  Good job @fredkingham made me add some tests :P